/root/mugen-ubuntu-riscv/libs/locallibs/common_lib.sh: line 86: rpm: command not found
/root/mugen-ubuntu-riscv/libs/locallibs/common_lib.sh: line 87: dnf: command not found
Thu Feb  9 19:14:55 2023 - INFO  - Start to prepare the test environment.
oe_test_chrony_chronyd.sh: line 24: [1: command not found
Traceback (most recent call last):
  File "/root/mugen-ubuntu-riscv/libs/locallibs/apt_manage.py", line 12, in <module>
    import ssh_cmd
  File "/root/mugen-ubuntu-riscv/libs/locallibs/ssh_cmd.py", line 23, in <module>
    import paramiko
ModuleNotFoundError: No module named 'paramiko'
Thu Feb  9 19:14:58 2023 - INFO  - End to prepare the test environment.
Thu Feb  9 19:14:59 2023 - INFO  - Start to run test.
Failed to start chronyd.service: Unit chronyd.service not found.
Unit chronyd.service could not be found.
Thu Feb  9 19:15:00 2023 - ERROR - oe_test_chrony_chronyd.sh line 36
Failed to get unit file state for chronyd.service: No such file or directory
Thu Feb  9 19:15:00 2023 - ERROR - oe_test_chrony_chronyd.sh line 38
oe_test_chrony_chronyd.sh: line 40: chronyc: command not found
Thu Feb  9 19:15:01 2023 - ERROR - oe_test_chrony_chronyd.sh line 41
Failed to stop chronyd.service: Unit chronyd.service not loaded.
Unit chronyd.service could not be found.
Thu Feb  9 19:15:02 2023 - ERROR - oe_test_chrony_chronyd.sh line 44
Thu Feb  9 19:15:02 2023 - INFO  - End to run test.
Thu Feb  9 19:15:03 2023 - ERROR - failed to execute the case.
Thu Feb  9 19:15:03 2023 - INFO  - Start to restore the test environment.
Thu Feb  9 19:15:04 2023 - WARN  - no thing to do.
Thu Feb  9 19:15:05 2023 - INFO  - End to restore the test environment.
